src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp

```diff
@@ -81,9 +99,17 @@ bool ov::intel_cpu::ACLInterpolateExecutorBuilder::isSupportedConfiguration(
         const std::vector<MemoryDescPtr> &dstDescs) {
     auto& inp_shape = srcDescs[0]->getShape().getDims();
     auto& out_shape = dstDescs[0]->getShape().getDims();
-
-    float scale_h = static_cast<float>(out_shape[2]) / inp_shape[2];
-    float scale_w = static_cast<float>(out_shape[3]) / inp_shape[3];
+    int index_h, index_w;
+    if (srcDescs[0]->hasLayoutType(LayoutType::ncsp)) {
+        index_h = 2;
+        index_w = 3;
```

Comment:
What about other ranks, i.e. rank 2?

Reply:
Discussed, can we add an assert for a rank here?

Reply:
ACL interpolate use only rank = 4